Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/features/settings/ui/SettingsAdapter.java

```diff
@@ -79,67 +84,68 @@ public SettingsAdapter(SettingsFragmentView view, Context context)
-        return new SingleChoiceViewHolder(view, this);
+        ListItemSettingBinding singleChoiceBinding = ListItemSettingBinding.inflate(inflater);
+        return new SingleChoiceViewHolder(singleChoiceBinding, this);
 
       case SettingsItem.TYPE_SLIDER:
-        view = inflater.inflate(R.layout.list_item_setting, parent, false);
-        return new SliderViewHolder(view, this, mContext);
+        ListItemSettingBinding sliderBinding = ListItemSettingBinding.inflate(inflater);
+        return new SliderViewHolder(sliderBinding, this, mContext);
 
       case SettingsItem.TYPE_SUBMENU:
-        view = inflater.inflate(R.layout.list_item_submenu, parent, false);
-        return new SubmenuViewHolder(view, this);
+        ListItemSubmenuBinding submenuBinding = ListItemSubmenuBinding.inflate(inflater);
+        return new SubmenuViewHolder(submenuBinding, this);
 
       case SettingsItem.TYPE_INPUT_BINDING:
-        view = inflater.inflate(R.layout.list_item_setting, parent, false);
-        return new InputBindingSettingViewHolder(view, this, mContext);
+        ListItemSettingBinding inputBinding = ListItemSettingBinding.inflate(inflater);
+        return new InputBindingSettingViewHolder(inputBinding, this, mContext);
 
       case SettingsItem.TYPE_RUMBLE_BINDING:
-        view = inflater.inflate(R.layout.list_item_setting, parent, false);
-        return new RumbleBindingViewHolder(view, this, mContext);
+        ListItemSettingBinding rumbleBinding = ListItemSettingBinding.inflate(inflater);
+        return new RumbleBindingViewHolder(rumbleBinding, this, mContext);
 
       case SettingsItem.TYPE_FILE_PICKER:
-        view = inflater.inflate(R.layout.list_item_setting, parent, false);
-        return new FilePickerViewHolder(view, this);
+        ListItemSettingBinding filePickerBinding = ListItemSettingBinding.inflate(inflater);
+        return new FilePickerViewHolder(filePickerBinding, this);
 
       case SettingsItem.TYPE_RUN_RUNNABLE:
-        view = inflater.inflate(R.layout.list_item_setting, parent, false);
-        return new RunRunnableViewHolder(view, this, mContext);
+        ListItemSettingBinding runnableBinding = ListItemSettingBinding.inflate(inflater);
+        return new RunRunnableViewHolder(runnableBinding, this, mContext);
 
       case SettingsItem.TYPE_STRING:
-        view = inflater.inflate(R.layout.list_item_setting, parent, false);
-        return new InputStringSettingViewHolder(view, this);
+        ListItemSettingBinding stringBinding = ListItemSettingBinding.inflate(inflater);
+        return new InputStringSettingViewHolder(stringBinding, this);
 
       case SettingsItem.TYPE_HYPERLINK_HEADER:
-        view = inflater.inflate(R.layout.list_item_header, parent, false);
-        return new HeaderHyperLinkViewHolder(view, this, mContext);
+        ListItemHeaderBinding hyperlinkBinding = ListItemHeaderBinding.inflate(inflater);
+        return new HeaderHyperLinkViewHolder(hyperlinkBinding, this);
```

Comment:
Perhaps consider doing it like this instead? (Including the other occurrences above)

```suggestion
        return new HeaderHyperLinkViewHolder(ListItemHeaderBinding.inflate(inflater), this);
```